storage/innobase/include/fsp0fsp.ic

```diff
@@ -366,11 +366,6 @@ inline bool fsp_flags_is_valid(uint32_t flags) {
     return (false);
   }
 
-<<<<<<< HEAD
-#if FSP_FLAGS_POS_UNUSED != 15
-#error You have added a new FSP_FLAG without adding a validation check.
-#endif
```

Comment:
This seems like a valid check. Is it removed by upstream? Did upstream add a new FSP flag?

Reply:
This comes from https://github.com/altmannmarcelo/percona-xtrabackup/commit/e001f2a533c4b8edd26e8e1723f00ed4a71d80c7 They removed `#defines` from code.
This particular part change to a `static_assert` - https://github.com/mysql/mysql-server/blob/mysql-8.0.29/storage/innobase/include/fsp0fsp.ic#L376

Reply:
ok, if there is static_assert, then we are all good